server/service/src/sync/README.md

```diff
@@ -1,3 +1,99 @@
+
+`active store`: A store can only belong to one site, even though data for any store is accessible on the central server, certain remote records that are owned by a store can only be edited on a site where the store is active. 
+
+`transfer`: Some data needs to be transferred between sites, for example a shipment from one site to another, we call these records transfers. See [processors](https://github.com/msupply-foundation/open-msupply/blob/bc83acbb3cd51fe3375ac01135c6eb880a793936/server/service/src/processors/README.md#L1) for more details
+
+## Record Types
+
+Records are treated differently by central server and remote sites based on data types, for example remote records belong to only one store and are only editable on a site where this store is active, whereas central data is only editable on a central server, the former usually syncs from remote site to central server whereas the later will only travel from central server to remote site.
+
+`remote record/data`: Can only be owned and edited by one store, on a site where this store is active, this data typically travels from remote site where it’s created and edited, to central server, which in turn may decide to forward it to another remote site (in case of transfers). Shipment, requisition, stock_line are examples of remote record.
+ 
+`central record/data`: This record is only editable on central server, and only travels in one direction, from central server to all remote sites. All central data will sync to all remote sites. Items and units are examples of central records.
+
+`central-remote record/data`: Similar in editability to central data, but will only sync to remote sites where these records are relevant. For example, master_list or name_store_join.
+
+`transfer record/data`: This is remote record but for a store that is not active on current site, this record is visible but not editable. For example, request requisition in response site.
+
+`shared-remote record/data`: Editable on any site and sync to all sites. For example barcodes
+
+`patient record/data`: Editable and accessible on any site where the patient is visible. Visibility in this case is determined by name store join. Prescribers, patients and patient documents are examples of patient record.
+
+
+## Phases/Stages of Synchronisation:
+
+`initialisation`: After the site is configured, it will first need to be initialised. This is where all relevant central and remote records are queued and sent to a remote site. Remote site is not ‘usable’ until initialisation stage is completed and all records are received
+
+`operational`: Once initialised the site will be in operational synchronisation phase, any changes to remote data will be sent to central server, and any changes to central data will travel on remote site from central server.
```

Comment:
Is this what you mean? I wasn't sure about the 'travel on'
```suggestion
`operational`: Once initialised the site will be in an operational synchronisation phase, any changes to remote data will be sent to central server, and any changes to central data will travel to the remote site from the central server.
```

Reply:
Yeah ! travel to, thanks